### PR TITLE
Docs - Contribution Triage: Formatted text and changed some titles and structure

### PR DESCRIPTION
**Pull request by juanmaguitar**
## What?
Highlighted some text with formatting and changed some titles and structure for this page. Also removed the reference to `Priority OMGWTFBBQ` which doesn't seem to be active anymore

## Why?
To improve readability and ease information ingestion

### CODE REVIEW

File: docs/contributors/triage.md [hunk 1]
```diff
@@ -80,21 +79,23 @@ Issues are closed for the following reasons:
...
 
 Along with the general triage flows listed previously, there are some specific additions to the flows for more design-centric triage for design minded folks participating in triage.
 
 -   PR testing and reviews: this should be your first stop for daily self triage.
--   Needs design feedback: check if the issue does need design feedback and, if possible, give it. You can organize this by priority, project boards or by least commented. Once there are enough opinions, please remove this label and decide on next steps (ie adding the Needs Design label).
--   Needs design: Does it really need a design? Does this fit a focus? If it has a design mark as ‘needs design feedback’ to better categorize the issue.
+-  Label `Needs design feedback`: check if the issue does need design feedback and, if possible, give it. You can organize this by priority, project boards or by least commented. Once there are enough opinions, please remove this label and decide on next steps (ie adding the Needs Design label).
+- Label  `Needs design`: Does it really need a design? Does this fit a focus? If it has a design mark as `Needs design feedback` to better categorize the issue.
```
Comment (ndiego) [inline docs/contributors/triage.md]: This is super minor, but the labels are all capitalized.

**Review by ndiego — APPROVED** [review]

File: docs/contributors/triage.md (hunk 1 above)
Comment (juanmaguitar) [reply docs/contributors/triage.md]: Thanks for the heads-up. 
Fixed! ✅

File: docs/contributors/triage.md [hunk 2]
```diff
@@ -18,33 +20,33 @@ These are the expectations of being a triage team member:
 
 If you would like to join this team, simply ask in #core-editor [Slack](https://make.wordpress.org/chat/) at any time.
 
-### Getting started
+## Triage on your own
```
Comment (annezazu) [inline docs/contributors/triage.md]: This doesn't feel as welcoming as "Getting started", which is fairly common for these kinds of docs. I might rename to something else or keep as is: "Triage your first issues" perhaps.

**Review by annezazu — APPROVED** [review]
Approving with just one optional comment to consider :)

File: docs/contributors/triage.md (hunk 2 above)
Comment (juanmaguitar) [reply docs/contributors/triage.md]: Yes, "Triage your first issues" sounds better 😊
Fixed!

**Pull request merged**